fuzzers/071-ppips/Makefile

```diff
@@ -24,10 +24,13 @@ $(SPECIMENS_OK):
 	touch $@
 
 run:
```

Comment:
Would it make sense to move run: and other common things into a common makefile?

Reply:
As I mentioned yesterday during the video conference, `make run` is not doing the same for all fuzzers.